cypress.config.js

```diff
@@ -0,0 +1,36 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  video: true,
+  chromeWebSecurity: true,
+  fixturesFolder: '.cypress/fixtures',
+  screenshotsFolder: '.cypress/screenshots',
+  videosFolder: '.cypress/videos',
+  downloadsFolder: '.cypress/downloads',
+  viewportWidth: 2000,
+  viewportHeight: 1320,
+  requestTimeout: 60000,
+  responseTimeout: 60000,
+  defaultCommandTimeout: 60000,
+  //experimentalNetworkStubbing: true,
+  //experimentalMemoryManagement: true,
+  numTestsKeptInMemory: 10, //Default value 50, chrome crashes without lowering
```

Comment:
"numTestsKeptInMemory: 10" implemented to prevent test browser from crashing.
<img width="889" alt="Screenshot 2023-05-15 at 2 16 42 PM" src="https://github.com/opensearch-project/dashboards-observability/assets/105462877/7f01c54f-8fc1-4768-a839-56fe35f36f8e">